### ISSUE TITLE
Rasterize issue

### ISSUE THREAD

**Issue by mapdevsaikat** [osmcode/osmium-contrib#10]
Hi joto,
 I'm running with the problem with rasterize. Could you please let me know how to resolve this?
The message is **./rasterize.sh: 20: .: Can't open out/bbox**

**Comment by joto** [comment]
Do you have a directory called `out`?

**Comment by mapdevsaikat** [comment]
Yes, I have. All the files I can see based on date. Inside out folder there
is one file is xbox. I am unable to run the restarize function after that.

**Comment by joto** [comment]
Sorry, that's not enough to go on for me. Can you write down exactly what commands you used?

**Comment by mapdevsaikat** [comment]
So, after setting up the `mapolution`, I have copied the *country.osh.pbf file inside `build` directory and run the code by with `./mapolution -s 2019-08-01 -e 2020-08-01 -S 5 *country.osh.pbf`. After running this line it is creating <out> folder and necessary files with 5 days interval with a bbox file.
Changed directory to <mapolution> and run this `./rasterize.sh` and getting the message `./rasterize.sh: 20: .: Can't open out/bbox`.
